homeassistant/components/zha/device_trigger.py

```diff
@@ -40,8 +43,10 @@
 async def async_attach_trigger(hass, config, action, automation_info):
```

Comment:
Should we start adding type hinting in the functions we touch?

Reply:
Will do on all subsequent PR's as I mentioned on Discord. I wanna get this fix in.